src/apps/btc/btc.h

```diff
@@ -49,6 +49,19 @@ USE_RESULT bool app_btc_xpub(
     char* out,
     size_t out_len);
 
+/**
+ * Returns the identifier xpub.
```

Comment:
"identifer xpub" does not describe anything really. this should mention electrum, the purpose, etc.